components/Footer.js

```diff
@@ -0,0 +1,58 @@
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import styled from 'styled-components'
+import {
+  faFacebook,
+  faInstagram,
+  faLinkedinIn,
+  faMediumM,
+  faTwitter,
+  faYoutube,
+} from '@fortawesome/free-brands-svg-icons'
+
+const FooterContainer = styled.div`
+  background: #3A396E;
+  color: #20FFAF;
+  text-align: center;
+  padding: 64px 0;
+`;
+
+const SocialMediaIcons = styled.div`
+  font-size: 32px;
+  margin-bottom: 32px;
+
+  a {
+    margin: 0 32px;
+
+    &:hover {
+      color: #FFFFFF;
+    }
+  }
+`;
+
+export default function Footer() {
```

Comment:
I think you can just do this since the name of the component is the same as the file name
```suggestion
export default () => {
```

Reply:
eslint got mad at me >:(
![image](https://user-images.githubusercontent.com/5078356/127420994-d259c2ac-daaf-4035-ab2d-9fea447114ac.png)

If we want to stick with arrow syntax then the change would be something like
```
const Footer = () => { ... }
export default Footer
```
but at that point I think we'd want to keep the original syntax. Thoughts?

Reply:
ahh ok lets just stick to named arrow functions. apparently anonymous arrow functions can cause fast refresh to not preserve local state 
```
Anonymous arrow functions like export default () => <div />; cause Fast Refresh to not preserve local component state. For large codebases you can use our name-default-component codemod.
```